README.md

```diff
@@ -14,9 +14,9 @@ setup.
 
 # Getting Ready
 
-If you have not already, please install [.NET 5.0](https://dotnet.microsoft.com/en-us/download/dotnet/5.0)
```

Comment:
.NET 7 is a [Standard Term Support release](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core) that ends support in four months, while .NET 8 is good until Nov 2026.
We should update both repos to .NET 8.

Reply:
Bumped it up to 8.0 and updated all dependencies